guides/welfare/parental_leave.md

```diff
@@ -1,7 +1,17 @@
-# Parental Leave
+# Maternity, Paternity & Shared Parental Leave
 
-Our current parental leave package is two weeks at full pay followed by statutory allowance.
+When you welcome a new child (birthed or adopted), Made Tech encourages you to take leave, to welcome the child into the world.
```

Comment:
I'd recommend `introducing your child to the family` or something, adopted children generally older so have already see the world a bit. Just a thought